src/components/TodoList.tsx

```diff
@@ -0,0 +1,53 @@
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+import React from 'react';
+
+export interface Props {
+  todos: Todo[];
+  VisibleTodos: Todo[];
```

Comment:
we usually name var in camelCase
```suggestion
  visibleTodos: Todo[];
```